config/vendored/cluster-api/crds/cluster_v1alpha1_machinedeployment.yaml

```diff
@@ -0,0 +1,120 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Does this PR include any tooling for updating these vendored CRDs? Would it make sense to pull them directly from the vendored version of cluster-api or create symlinks there instead?

Reply:
We don't commit the vendor directory, unlike the GCP provider.

I'm not sure if we should expect people following the getting started directions to do a dep ensure? I guess not? Would appreciate a 2nd opinion.

Will add a copy command to update the vendored directory in Makefile. Not sure what the correct invocation in Bazel is though.

Reply:
From the notes above:
> During bootstrap, the user is instructed to use clusterawsadm to copy AWS credentials into Minikube under /home/docker/.aws/credentials which is then added as a
volume mount to the statefulset as /root/.aws.

> As no credentials are provided in the statefulset, the SDK will look for the shared credentials file, find it in the default path and load from there.

> After pivot, the new control plane will create an empty /home/docker/.aws directory, but because the user has run the CloudFormation bootstrap and our sample manifests specify the created IAM role, the controller will fall through to checking EC2 instance metadata and find credentials there to continue.

For reference, the Java SDK explains this best https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html#credentials-default

Reply:
Considering that we've run into issues related to vendor not being checked in, I think I'm quite ok with biting the bullet and just checking vendor in. We'll probably want to do that as a separate independent PR, though :)

Reply:
OK, I'll leave it as is, and then add vendor in another PR.